master/static/migrations/20240829000038_add-task-config-policy-table.tx.up.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TYPE workload_type AS ENUM ('EXPERIMENT', 'NTSC');
+
+CREATE TABLE IF NOT EXISTS task_config_policies
+(
+    workspace_id integer,  
+    workload_type workload_type NOT NULL,
+    last_updated_by integer NOT NULL,
+    last_updated_time timestamptz NOT NULL DEFAULT current_timestamp,
+    invariant_config jsonb,
+    constraints jsonb,
+
+    CONSTRAINT fk_wksp_id FOREIGN KEY(workspace_id) REFERENCES workspaces(id) ON DELETE CASCADE,
+    CONSTRAINT fk_last_updated_by_user_id FOREIGN KEY(last_updated_by) REFERENCES users(id)
+        ON DELETE CASCADE
```

Comment:
This means that if a user is deleted, the config policies they created would also be deleted? AFAIK we don't delete users, but is that the behavior we want?

Reply:
I agree, I think we should check with product on this

Reply:
Hm! Thaat's a good point. 
Probably doesn't make sense to remove a TCP just because the user who created it is deleted. And like you said, I don't even think we delete users, we just activate/deactivate them. 
I removed the on delete cascade!